pandas/core/reshape.py

```diff
@@ -1079,8 +1079,12 @@ def get_dummies(data, prefix=None, prefix_sep='_', dummy_na=False,
     prefix_sep : string, default '_'
         If appending prefix, separator/delimiter to use. Or pass a
         list or dictionary as with `prefix.`
-    dummy_na : bool, default False
-        Add a column to indicate NaNs, if False NaNs are ignored.
+    dummy_na : bool or 'keep', default False
+        If True, then add a column to indicate NaNs otherwise if
+        False NaNs are ignored. If set to 'keep', NaNs will be propagated
+        into the final result.
+
+        .. versionadded:: 0.20.0
     columns : list-like, default None
```

Comment:
I am confused. Here are the other docs in this function.

<img width="540" alt="screen shot 2017-04-07 at 8 58 26 am" src="https://cloud.githubusercontent.com/assets/5296416/24804683/dd348676-1b74-11e7-8a85-e5879a17e34b.png">

You can see I followed what was done in the other cases. Do these other ones need to be changed too? How did the docs compile at all?